.github/actions/spelling/reject.txt

```diff
@@ -0,0 +1,22 @@
+^attache$
+^attacher$
+^attachers$
+^spae$
+^spaebook$
+^spaecraft$
+^spaed$
+^spaedom$
+^spaeing$
+^spaeings$
+^spae-man$
```

Comment:
Technically this isn't necessary, as the bot sees this as `spae` + `man`, and `spae` is already removed on line 4 above and `man` is a word, so `spae-man` in a file would be flagged for `spae`.